docs/authentication.md

```diff
@@ -112,6 +112,26 @@ When using `-AccessToken`, we won't have access to the refresh token and the cli
 
```

Comment:
> Having reviewed this PR, it has 2 very big problems:
> 
> * it is making un-authenticated request to graph every time a token is needed, to fetch a new nonce. **This will double the HTTP calls to Graph !** This is not needed - a new nonce should be retrieved from 200 OK response Authenticate-Info; Even if this is not available, it is better to reuse old nonce, it has a few min validity.
> * it is not taking into consideration that authenticated request with POP token can return 401 + invalid nonce response (e.g. nonce expired), in which case a new nonce must be retrieved.
> 
> For a prototype implementation of the protocol, please see https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=/PoPTokensProtocol/PoP_API_In_MSAL.md&_a=preview&version=GBdev&anchor=appendix---how-applications-can-use-this-lower-level-api

Need to check that the `Autheticate-Info` is supported by MS Graph.